package.json

```diff
@@ -55,6 +55,7 @@
   },
   "devDependencies": {
     "@types/node": "20.8.2",
+    "@typescript-eslint/eslint-plugin": "^7.4.0",
```

Comment:
Adding more dev helpers should be in a separate PR with an explanation of why this would be useful.